src/compile/data/aggregate.ts

```diff
@@ -117,7 +117,8 @@ export class AggregateNode extends DataFlowNode {
         } else {
           meas[s.field] = meas[s.field] || {};
           meas[s.field][s.aggregate] = true;
-          as[s.field] = s.as;
+          as[s.field] = as[s.field] || {};
```

Comment:
For summarize transform, automatically add `as` if `as` is not defined (using `||`)